torchvision/ops/misc.py

```diff
@@ -97,13 +84,16 @@ def __init__(
         dilation: int = 1,
         inplace: Optional[bool] = True,
         bias: Optional[bool] = None,
+        conv_layer: Callable[..., torch.nn.Module] = torch.nn.Conv2d,
     ) -> None:
+
```

Comment:
I believe we can detect it if someone instantiated `ConvNormActivation` directly and throw a warning.

The way to do this is by something like:
```python
if self.__class__ == ConvNormActivation:
    warn("Please dont call me directly ....")
```

Reply:
Nice trick! I didn't know this. Works like charm. Fixed it!